<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_moshakhasats_books_by_manager</class>
 <widget class="QDialog" name="Dialog_moshakhasats_books_by_manager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>812</width>
    <height>836</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>IranNastaliq</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>730</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>IranNastaliq</family>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="text">
    <string>بعدی</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>730</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>IranNastaliq</family>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="text">
    <string>بازگشت</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>IranNastaliq</family>
     <pointsize>23</pointsize>
     <italic>false</italic>
    </font>
   </property>
   <property name="text">
    <string> نام کتاب</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>80</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>IranNastaliq</family>
     <pointsize>23</pointsize>
     <italic>false</italic>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>موضوع کتاب</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>140</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>IranNastaliq</family>
     <pointsize>23</pointsize>
     <italic>false</italic>
    </font>
   </property>
   <property name="text">
    <string> انتشارات کتاب</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>IranNastaliq</family>
     <pointsize>23</pointsize>
     <italic>false</italic>
    </font>
   </property>
   <property name="text">
    <string>  شماره ملی اهدا کننده کتاب</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>330</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>IranNastaliq</family>
     <pointsize>24</pointsize>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>    نویسنده کتاب</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>395</y>
     <width>55</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>IranNastaliq</family>
     <pointsize>23</pointsize>
     <italic>false</italic>
    </font>
   </property>
   <property name="text">
    <string>  نام</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>400</y>
     <width>81</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>IranNastaliq</family>
     <pointsize>23</pointsize>
     <italic>false</italic>
    </font>
   </property>
   <property name="text">
    <string> نام خانوادگی</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>520</y>
     <width>81</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>IranNastaliq</family>
     <pointsize>23</pointsize>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>  مترجم</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>560</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>IranNastaliq</family>
     <pointsize>23</pointsize>
    </font>
   </property>
   <property name="text">
    <string>نام خانوادگی</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>570</y>
     <width>55</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>IranNastaliq</family>
     <pointsize>23</pointsize>
    </font>
   </property>
   <property name="text">
    <string>  نام</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>390</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>IranNastaliq</family>
     <pointsize>23</pointsize>
     <italic>false</italic>
    </font>
   </property>
   <property name="text">
    <string>     ملیت</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>260</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>IranNastaliq</family>
     <pointsize>23</pointsize>
     <italic>false</italic>
    </font>
   </property>
   <property name="text">
    <string>      تعداد کتاب</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="name">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>40</y>
     <width>181</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(192, 241, 203, 255), stop:1 rgba(255, 255, 255, 255));</string>
   </property>
   <property name="placeholderText">
    <string>نام کتاب را وارد نمایید</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="subject">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>100</y>
     <width>181</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(192, 241, 203, 255), stop:1 rgba(255, 255, 255, 255));</string>
   </property>
   <property name="placeholderText">
    <string>موضوع کتاب را وارد نمایید</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="entesharat">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>150</y>
     <width>181</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(192, 241, 203, 255), stop:1 rgba(255, 255, 255, 255));</string>
   </property>
   <property name="placeholderText">
    <string>انتشارات کتاب را وارد نمایید</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ehdakonande">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>210</y>
     <width>181</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(192, 241, 203, 255), stop:1 rgba(255, 255, 255, 255));</string>
   </property>
   <property name="placeholderText">
    <string>شماره ملی اهداگرراواردنمایید</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="write_last">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>460</y>
     <width>181</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(192, 241, 203, 255), stop:1 rgba(255, 255, 255, 255));</string>
   </property>
   <property name="placeholderText">
    <string>نام خانوادگی نویسنده بنویسید</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="write_name">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>460</y>
     <width>181</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(192, 241, 203, 255), stop:1 rgba(255, 255, 255, 255));</string>
   </property>
   <property name="placeholderText">
    <string>نام نوینسده را وارد نمایید</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="write_national">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>460</y>
     <width>181</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(192, 241, 203, 255), stop:1 rgba(255, 255, 255, 255));</string>
   </property>
   <property name="placeholderText">
    <string>ملیت نوینسده را وارد نمایید</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="translat_name">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>660</y>
     <width>181</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(192, 241, 203, 255), stop:1 rgba(255, 255, 255, 255));</string>
   </property>
   <property name="placeholderText">
    <string>نام مترجم را وارد نمایید</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="translat_family">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>660</y>
     <width>181</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(192, 241, 203, 255), stop:1 rgba(255, 255, 255, 255));</string>
   </property>
   <property name="placeholderText">
    <string>نام خانوادگی مترجم بنویسید</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="number">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>270</y>
     <width>181</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(192, 241, 203, 255), stop:1 rgba(255, 255, 255, 255));</string>
   </property>
   <property name="placeholderText">
    <string>تعداد کتاب مدنظر را وارد نمایید</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>180</y>
     <width>341</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>IranNastaliq</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>             اگر کتاب خریداری شده است 10 عدد صفر وارد کنید</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
